GIT_CONVENTIONS.md

```diff
@@ -0,0 +1,87 @@
+# Git Conventions for the ORAS Community
+
+This guide is a specification for adding human and machine readable meaning to Pull Request and git commit messages in the ORAS community. It provides an easy set of rules for creating an explicit commit history; which makes it easier to explore a more structured commit history and automatically generating CHANGELOGs for each new release. 
```

Comment:
```suggestion
This guide is a specification for adding human and machine readable meaning to Pull Request and git commit messages in the ORAS community. It provides an easy set of rules for creating an explicit commit history that makes it easier to explore a more structured commit history and automatically generating CHANGELOGs for each new release. 
```
If you want to use which, the pattern is usually `<statement1>, which <statement2>.` When you want to use a semicolon `;` both the first and second sentence need to be complete sentences. My (personal opinion) is that it's more straight forward to use `that`.